docs/ruleset.rst

```diff
@@ -43,6 +43,14 @@ Left-hand sides are currently a few special forms which select nodes to be fed t
 
 .. autofunction:: and(typeCall[, typeCall, ...])
 
+.. function :: when(predicate)
+
+    Further specify types of node you'd like to select. Can be chained with type() or dom().
+
+    Example: dom('p').when(fnode => fnode.element.id === 'foo')
```

Comment:
And you can stick double-backticks around the code to make it show up monospaced in the docs.